site/src/patterns/form-validation.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Form Validation
+description: The Form validation pattern in the Atlas Design System
+template: standard
+---
+
+Use the form behavior component within a form to ensure accessible form validation. It expands on HTML5 validation.
+
+```html
+<form id="question-ask" data-form-type="question" action="#" method="POST" new="" novalidate="">
```

Comment:
Currently using this as a POC to verify things will still work for QnA, so the examples here are taken from QnA ask question form template.